Packs/HarfangLabEDR/Integrations/Hurukai/Hurukai.py

```diff
@@ -0,0 +1,2284 @@
+                technique_id = []
+
+                for tag in tags:
+                    if tag.startswith('attack'):
+                        content = tag[7:]
+                        if content in TACTICS:
+                            tactic.append(TACTICS[content])
+                        elif content[0] == 't':
+                            technique_id.append(content)
+
+                incident = {
+                    'name': alert.get('rule_name', None),
+                    'occurred': alert.get('alert_time', None),
+                    'severity': SEVERITIES.index(alert.get('level', '').capitalize()) + 1,
+                    'rawJSON': json.dumps(alert)
+                }
+
+                alert_id = alert.get('id', None)
+                if alert_id not in already_fetched_previous:
+                    incidents.append(incident)
+                    already_fetched_current.append(alert_id)
+
+                if incident_created_time_us > latest_created_time_us:
+                    latest_created_time_us = incident_created_time_us
+
+                total_number_of_alerts += 1
+                if max_results and total_number_of_alerts >= max_results:
+                    break
+
+        args['offset'] += len(results['results'])
+        if results['count'] == 0 or not results['next'] or (max_results and total_number_of_alerts >= max_results):
+            break
+
+    next_run = {'last_fetch': latest_created_time_us, 'already_fetched': already_fetched_current}
+
+    demisto.setLastRun(next_run)
+    demisto.incidents(incidents)
+
+    return next_run, incidents
+
+
+def get_endpoint_info(client, args):
+    agent_id = args.get('agent_id', None)
+
+    agent = client.get_endpoint_info(agent_id)
+
+    readable_output = tableToMarkdown(f'Endpoint information for agent_id : {agent_id}', agent)
+
+    outputs = {
+        'Harfanglab.Agent(val.agentid == obj.agentid)': agent
+    }
+
+    return_outputs(
+        readable_output,
+        outputs,
+        agent
+    )
+    return agent
+
+
+def endpoint_search(client, args):
+    hostname = args.get('hostname', None)
+
+    data = client.endpoint_search(hostname)
+
+    readable_output = tableToMarkdown(f'Endpoint information for Hostname : {hostname}', data['results'])
+
+    outputs = {
+        'Harfanglab.Agent(val.agentid == obj.agentid)': data['results']
+    }
+
+    return_outputs(
+        readable_output,
+        outputs,
+        data
+    )
+    return data
+
+
+def job_create(client, args, parameters=None, can_use_previous_job=True):
+    action = args.get('action', None)
+    agent_id = args.get('agent_id', None)
+
+    if action is None or agent_id is None:
+        return False, None
+
+    if can_use_previous_job:
+        previous_job_id = find_previous_job(client, action, agent_id)
+        if previous_job_id is not None:
+            return True, previous_job_id
+
+    data = client.job_create(agent_id, action, parameters)
+    time.sleep(2)
```

Comment:
Please remove this.

Reply:
Done.

Reply:
1. Generic polling : we will update it in the next version of the connector.
2. Changes to .circleci/config.yml : Which changes are you talking about ?
3. URL concatenation : done (thanks for the tip !)
4. README generation: done
5. Release notes for the common incident field: done
6. removeNull=True: done
Thanks !